redhat-cop/uncontained.io
# Add favicon and apple touch icon.

#### What is this PR About?
Adds a favicon and a touch icon.

#### How should we test or review this PR?
Run the site locally. Confirm that the favicon appears in the tab.

Optionally, connect to the site from a mobile device. Add the site to your home screen. Confirm that the touch icon appears.

#### Is there a relevant Trello card or Github issue open for this?
Resolves #90.

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
